app/utils/pptx_lecture_ingest.py

```
@@ -0,0 +1,51 @@
+from pathlib import Path
+import json
+from pptx_text_extractor import extract_pptx_to_chunks
+
+
+def categorize_pptx(path: Path) -> str:
+    """Heuristic categorization based on folder/file names."""
+    parts = [p.lower() for p in path.parts]
+    name = path.name.lower()
```

Ruff (0.15.2)

[error] Local variable `name` is assigned to but never used

Remove assignment to unused variable `name`

(F841)